examples/open-deep-research/agent.py

```diff
@@ -122,11 +121,8 @@
     inputs=[chatui_client],
     outputs=[chatui_client],
     agent=manager_agent,
-    memory_repository=MemoryRepository(
-        api_key=os.getenv("OPENAI_API_KEY"),
-        agent_name="open_deep_research_agent",
-        short_term_memory_limit=4,
-    ),
+    use_long_term_memory=True,
```

Comment:
Similar to above - there should be a single, encapsulated memory `Memory` object which configures all memory aspects.